src/CustomDatePickerIOS.js

```diff
@@ -63,10 +63,10 @@ export default class CustomDatePickerIOS extends React.PureComponent {
     minuteInterval: this.props.minuteInterval || 1
   };
 
-  componentWillReceiveProps(nextProps) {
-    if (this.props.date.valueOf() !== nextProps.date.valueOf()) {
+  componentDidUpdate = prevProps => {
```

Comment:
Moving `componentWillReceiveProps` to `componentDidUpdate` will cause an additional render that is not needed